packages/desktop-client/src/global-events.ts

```diff
@@ -163,6 +162,6 @@ export function handleGlobalEvents(store: AppStore) {
   });
 
   listen('api-fetch-redirected', () => {
-    store.dispatch(reloadApp());
+    window.Actual.reload();
```

Comment:
`reloadApp` just calls `window.Actual.reload()` so just call it directly instead of going through redux dispatch